wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -423,14 +423,15 @@ public final class MLSService: MLSServiceInterface {
 
         do {
             let config = ConversationConfiguration(
-                ciphersuite: CiphersuiteName.mls128Dhkemx25519Aes128gcmSha256Ed25519.rawValue,
+                ciphersuite: CiphersuiteName.default.rawValue,
                 externalSenders: backendPublicKeys.ed25519Keys,
-                custom: .init(keyRotationSpan: nil, wirePolicy: nil)
+                custom: .init(keyRotationSpan: nil, wirePolicy: nil),
+                perDomainTrustAnchors: [PerDomainTrustAnchor.init(domainName: "", intermediateCertificateChain: "")] // TODO: review this
```

Comment:
What does this TODO mean?
I assume leaving it as is has the same effect as removing it :)

Reply:
I didn't add this :) but trust anchors is something which comes with end2end identify. I think we can remove it.

Reply:
Remove it and now also don't configure any trust anchors at all.